bundle/render/render_text_output_test.go

```diff
@@ -326,10 +331,10 @@ func TestRenderTextOutput(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			writer := &bytes.Buffer{}
 
-			err := RenderTextOutput(writer, tc.bundle, tc.diags, tc.opts)
+			err := RenderDiagnostics(writer, tc.bundle, tc.diags, tc.opts)
 			require.NoError(t, err)
 
-			assert.Equal(t, tc.expected, writer.String())
+			require.Equal(t, tc.expected, writer.String())
```

Comment:
The rubric: use `require` when the test must immediately fail (i.e. on an error check), use `assert` elsewhere.

This is OK as is.